src/pages/blog/write/index.module.scss

```diff
@@ -0,0 +1,236 @@
+    border-bottom: 1px solid #e0e0e0;
+
+    @include for-mobile-all {
+      position: initial;
+      display: block;
+      height: 40px;
+      text-align: center;
+    }
+
+    a{
+      img{
+        width: 45px;
+        height: 45px;
+        object-fit: contain;
+      }
+
+      @include for-mobile-all {
+        display: block;
+        padding: 5px 0;
+
+        img{
+          width: 30px;
+          height: 30px;
+          vertical-align: middle;
+        }
+      }
+    }
+
+    .btnGroup{
+      display: flex;
+
+      @include for-mobile-all {
+        position: fixed;
+        width: 100%;
+        left: 0;
+        bottom: 0px;
+        z-index: 1;
+        padding: 10px 0;
+        border-top: 1px solid #e0e0e0;
+        background-color: rgba(255, 255, 255, .8);
+        justify-content: space-around;
+      }
+
+      button{
+        svg{
+          margin-right: 10px;
+        }
+
+        @include for-mobile-all {
+          margin: 0 5px !important;
+          font-size: 12px;
+          padding: 4px 6px;
+
+          svg{
+            margin-right: 5px;
+          }
+        }
+      }
+      
+      button:nth-child(1) {
+        margin:0 30px;
+      }
+
+      button:last-child {
+        color: white;
+      }
+    }
+  }
+
+  .main{
+    --height: calc(100vh - 60px - 40px);
+    max-width: 90vw;
+    margin: 20px auto;
+    min-height: var(--height);
+    border-radius: 8px;
+    box-shadow: 0px 0px 3px #ccc;
+
+    @include for-mobile-all {
+      // height = Screen height - Header height - Footer height - Title height - Editor margin
+      --height: calc(100vh - 40px - 50px - 40px - 20px);
+      max-width: 95vw;
+      margin: 10px auto;
+      height: var(--height);
+    }
+
+    .title{
+      flex: 1 auto;
+      background-color: #fafbfc;
+      border: 1px solid #e0e0e0;
+      border-bottom: 0;
+      border-top-left-radius: 8px;
+      border-top-right-radius: 8px;
+      height: 60px;
+
+      @include for-mobile-all {
+        height: 39px;
+      }
+
+      :global{
+        .MuiOutlinedInput-root,
+        .MuiOutlinedInput-notchedOutline,
+        input {
+          border: none;
+        }
+
+        input{
+          font-size: 26px;
+          font-weight: bold;
+          padding: 10px;
+
+          @include for-mobile-all {
+            font-size: 20px;
+            padding:5px 10px;
+          }
+        }
+      }
+    }
+
+    .editor{
+      min-height: calc(var(--height) - 60px);
+      border-bottom-left-radius: 8px;
+      border-bottom-right-radius: 8px;
+      overflow: hidden;
+
+      @include for-mobile-all {
+        width: 100%;
+        height: var(--height);
+      }
+    } 
+  }
+
+  :global{
+    .rc-md-editor{
+      .rc-md-navigation{
+        background-color: #fafbfc;
+
+        .button{
+          margin-right: 10px;
+        }
+
+        i{
+          color: #333;
+        }
+      }
+
+      .editor-container>.section{
+        border-right: none;
```

Comment:
if this border is removed, in my MacOS machine, the divide line is not visible since the new divide line in your new design is a vertical scrollbar. the scrollbar works fine under windows, but on Mac the scrollbar is only visible when you have something to scroll and when you are scrolling it

Reply:
could you find a way to fix this problem?

Reply:
<img width="438" alt="image" src="https://user-images.githubusercontent.com/105776364/229394247-901e761a-1c73-476e-a1a1-48b36fae7d99.png"> turns out there is a option to configure this on Mac. I think we can remove this line `border-right: none;` to make it more compatible on user end.